htdocs/core/class/notify.class.php

```diff
@@ -577,30 +577,58 @@ public function send($notifcode, $object, $filename_list = array(), $mimetype_li
 						$pdf_path = $dir_output."/".$ref.".pdf";
 						if (!dol_is_file($pdf_path)) {
 							// We can't add PDF as it is not generated yet.
-							$filepdf = '';
-						} else {
+							if (empty($conf->global->MAIN_DISABLE_PDF_AUTOUPDATE)) {
+								$outputlangs = $langs;
+								$newlang = '';
+
+								if ($conf->global->MAIN_MULTILANGS && empty($newlang)) {
+										  $newlang = $object->thirdparty->default_lang;
+								}
+								if (!empty($newlang)) {
+									$outputlangs = new Translate("", $conf);
+									$outputlangs->setDefaultLang($newlang);
+								}
+								$modelpdf = !empty($object->modelpdf)?$object->modelpdf:$conf->global->FACTURE_ADDON_PDF;
```

Comment:
Doc will not always be an invoice here. So use of $conf->global->FACTURE_ADDON_PDF must be done only if object->element is 'facture'
You must also deal with the case modelpdf is empty. You can just exclude generation of doc in such a case.

Reply:
fix done